Prog/TOPA/TOPA_CBO_spreadsheet.sas

```diff
@@ -0,0 +1,58 @@
+/**************************************************************************
+ Program:  TOPA_CBO_TA_spreadsheet.sas
+ Library:  PresCat
+ Project:  Urban-Greater DC
+ Author:   Elizabeth Burton
+ Created:  3/30/2023
+ Version:  SAS 9.4
+ Environment:  Local Windows session (desktop)
+ 
+ Description:  Create CBO spreadsheet for TA information
+ 
+ Modifications:
+**************************************************************************/
+
+%include "\\sas1\DCdata\SAS\Inc\StdLocal.sas";
+
+** Define libraries **;
+%DCData_lib( PresCat )
+
+%File_info( data=PresCat.TOPA_database, printobs=5 ) 
+%File_info( data=PresCat.TOPA_notices_sales, printobs=5 ) 
+
+/** Fill in missing columns from original TOPA database to notices sale database created**/
+data Topa_CBO_sheet; 
+  merge 
+    PresCat.TOPA_notices_sales (keep=id u_address_id_ref u_dedup_notice u_notice_date u_ownername u_sale_date in=time_period)
+    Prescat.Topa_database (keep=id All_street_addresses Property_name Date_DHCD_received_TA_reg Notes Technical_assistance_provider Tech_Assist_Staff Tenant_Assn_Lawyer Did_a_TA_claim_TOPA_rights TA_Development_Partner Date_TA_assignment_of_rights Developer_assignment_of_rights);
```

Comment:
@eburton9 I added a new variable, u_Date_DHCD_received_TA_reg, to TOPA_database. You should use that instead of Date_DHCD_received_TA_reg.